### PR TITLE
feat: add segregated witness structure

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

**Comment by duanyytop** [comment]
When will it be finished ？

**Comment by BaojunCZ** [comment]
I have no idea.I am following https://github.com/nervosnetwork/ckb-sdk-swift/pull/44

**Comment by duanyytop** [comment]
OK

**Comment by ashchan** [comment]
No need to hurry. Just implement and wait for CKB upstream to merge this feature.

File: console/src/main/java/org/nervos/ckb/APIClient.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public static void main(String[] args) throws IOException {
 
         System.out.println("CellOutputWithOutPoint: " + gson.toJson(RpcRequest.getLiveCell()));
 
-        System.out.println("Transaction hash: " + RpcRequest.sendTransaction(new Transaction(0, Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
+        System.out.println("Transaction hash: " + RpcRequest.sendTransaction(new Transaction(0, Collections.emptyList(), Collections.emptyList(), Collections.emptyList(), Collections.emptyList())));
```
Comment (ashchan) [inline console/src/main/java/org/nervos/ckb/APIClient.java]: @duanyytop Can we extract these test print outs from `main` by moving them to separate tests or method that a test/user can call explicitly?

Comment (duanyytop) [reply console/src/main/java/org/nervos/ckb/APIClient.java]: Now there is a test file named APITest.java, so APIClient is unnecessary.

Please @BaojunCZ delete these print outs code.

Comment (BaojunCZ) [reply console/src/main/java/org/nervos/ckb/APIClient.java]: ok

**Pull request merged**